.gitlab-ci.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+before_script:
+  - ln -s /var/lib/fc-ci/vmlinux.bin testdata/vmlinux
+  - ln -s /usr/local/bin/firecracker testdata/firecracker
+  - ln -s /usr/local/bin/jailer testdata/jailer
```

Comment:
It feels weird to have to use symlinks to get the right artifacts in place. We should consider updating our tests to allow specification of their location via environment variables or a config file. Not a blocker, though.